pyrasterframes/src/main/python/docs/raster-read.pymd

```diff
@@ -56,19 +58,19 @@ RasterFrames relies on three different IO drivers, selected based on a combinati
 
 | Prefix              | GDAL        | Java I/O | Hadoop   |
 | ------------------- | ----------- | -------- | -------- |
-| `gdal://<vsidrv>//` | +    | -        | -        |
-| `file://`           | +    | + | -        |
-| `http://`           | +    | + | -        |
-| `https://`          | +    | + | -        |
-| `ftp://`            | `/vsicurl/` | + | -        |
-| `hdfs://`           | `/vsihdfs/` |-         | + |
-| `s3://`             |  `/vsis3/`  | + | -        |
-| `s3n://`            |  -          | -        | + |
-| `s3a://`            |  -          | -        | + |
-| `wasb://`           |  `/vsiaz/`  | -        | + |
-| `wasbs://`          |  -          | -        | + |
+| `gdal://<vsidrv>//` | yes         | no       | no       |
+| `file://`           | yes         | yes      | no       |
+| `http://`           | yes         | yes      | no       |
+| `https://`          | yes         | yes      | no       |
+| `ftp://`            | `/vsicurl/` | yes      | no       |
+| `hdfs://`           | `/vsihdfs/` | no       | yes      |
+| `s3://`             | `/vsis3/`   | yes      | no       |
+| `s3n://`            | no          | no       | yes      |
+| `s3a://`            | no          | no       | yes      |
+| `wasb://`           | `/vsiaz/`   | no       | yes      |
+| `wasbs://`          | no          | no       | yes      |
```

Comment:
👍 FYI: The unicode characters caused problems with (LaTeX) PDF generation.

Reply:
yes and i think the use of `+` and `-` broke the table formatting in paradox or someplace